fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java

```diff
@@ -91,6 +95,24 @@ public interface AbstractBlockSettingsAccessor {
 	@Accessor
 	boolean isToolRequired();
 
+	@Accessor
```

Comment:
Since they aren't being used internally and are there in order to maintain compat with cursed mods, these new accessors need a `@Deprecated` mark